custom/conf/app.ini.sample

```diff
@@ -636,6 +636,8 @@ SCHEDULE = @every 24h
 UPDATE_EXISTING = true
 
 [git]
+; The path of git executable. If empty, Gitea searches through the PATH environment.
```

Comment:
Could you also add this to the cheatsheet in docs/ ?

Reply:
Which doc are you referring to? I don't find a file named 'cheatsheet'.

Reply:
https://github.com/go-gitea/gitea/blob/master/docs/content/doc/advanced/config-cheat-sheet.en-us.md

Reply:
Thx for posting whole link :)

Reply:
Done.